specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2015-05-01/components_API.json

```diff
@@ -486,6 +486,11 @@
           "externalDocs": {
             "url": "https://aka.ms/sl0yqd"
           }
+        },
+        "RetentionInDays": {
```

Comment:
Adding a writeable property within an API version is considered a breaking change. Why is this not going in a new version?

Reply:
Hi Kris
please refer to my update PR. Created new one with suitable name.

https://github.com/Azure/azure-rest-api-specs/pull/7539